populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
this file is using the service file which is also used by other scripts. If I move this into the course_discovery folder I'm not sure how I can import that service (other than maybe using __import__ but this seems less hacky than that)